src/java/org/apache/cassandra/service/accord/AccordCommandStore.java

```diff
@@ -572,4 +576,29 @@ static SafeRedundantBefore max(SafeRedundantBefore a, SafeRedundantBefore b)
             return a.ticket >= b.ticket ? a : b;
         }
     }
+
+    private @Nullable TableMetadata tableMetadata()
+    {
+        TableMetadataRef metadataRef = this.metadata;
+        if (metadataRef != null)
+            return metadataRef.get();
+
+        TableMetadata metadata = Schema.instance.getTableMetadata(tableId);
+        if (metadata == null)
+            return null;
+        this.metadata = metadata.ref;
+        return metadata;
+    }
+
+    @Override
+    public String toString()
+    {
+        TableMetadata metadata = tableMetadata();
+        StringBuilder sb = new StringBuilder("[");
+        if (metadata != null)
+            sb.append(metadata.keyspace).append('.').append(metadata.name).append('|');
```

Comment:
```suggestion
            sb.append(metadata).append('|');
```

`TableMetadata.toString` does this, but its valid CQL so the keyspace/name might be quoted if its needed